assets/scripts/streets/StreetMetaAnalytics.jsx

```diff
@@ -20,7 +20,7 @@ function StreetMetaAnalytics (props) {
   const averageTotal = getStreetCapacity(street, locale).averageTotal
 
   // For zero capacity, don't display anything
-  return (averageTotal > 0) && (
+  return (Number.parseInt(averageTotal) > 0) && (
```

Comment:
Include radix argument

Reply:
Oh interesting, I thought this was supposed to be caught by the linter, but `standard` relaxed this rule some time ago. https://github.com/standard/standard/issues/384

Reply:
I'm leaving this comment because I'm curious whether you think we should enforce for consistency or leave alone.

Reply:
Yeah, it doesn't technically affect this case but we would have some unexpected behavior down the road if we make it a habit. ✂️